<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KuponBruteForceForm</class>
 <widget class="QWidget" name="KuponBruteForceForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>396</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Kupon Brute Force</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>411</width>
     <height>371</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Kuponlar</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QListWidget" name="kupon_liste"/>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="kupon_ekle_button">
      <property name="text">
       <string>Kupon Ekle</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="kupon_sil_button">
      <property name="text">
       <string>Seçili Kuponu Sil</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="kupon_liste_temizle_button">
      <property name="text">
       <string>Temizle</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="kupon_liste_save_button">
      <property name="text">
       <string>Kuponları Kaydet</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="kupon_baslat_button">
      <property name="text">
       <string>Başlat</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="kupon_yukle_button">
      <property name="text">
       <string>Kupon Yükle (txt)</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
